cmd/crc/cmd/root.go

```diff
@@ -115,6 +115,8 @@ func Execute() {
 	runPostrun()
 }
 
+//Deprecated: checkIfMachineMissing is deprecated.
+// checkIfMachineMissingWithDaemon should be used
```

Comment:
Some of the commands are still using it so not deprecated completely right?

Reply:
Yes, I'm going to delete `checkIfMachineMissing` after finishing moving cmd to daemon